.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        python-version:
+          - 3.5
+          - 3.8
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: pip install -r requirements/ci.txt
+
+    - name: Run Tests
+      run: make quality
+
+    - name: Run Coverage
+      if: matrix.python-version == '3.8' && matrix.toxenv=='django32'
```

Comment:
Does the matrix have a `toxenv` setting?

Reply:
Removed the unnecessary condition.